### PR TITLE
Add veBoostV2 deployment

### PR DESCRIPTION
**Pull request by TomAFrench**
# Description

Adds a new deployment for the veBoostV2 contract. I've added fork tests that ensure that we can migrate all the existing boosts and that the adjusted balances of these accounts are unchanged by the migration.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- [x] Complex code has been commented, including external interfaces
- [x] Tests are included for all code paths
- [x] The base branch is either `master`, or there's a description of how to merge

### CODE REVIEW

File: pkg/deployments/tasks/20221205-veboost-v2/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# 2022-12-05 - veBoost V2
+
```
Comment (nventuro) [inline pkg/deployments/tasks/20221205-veboost-v2/readme.md:2]: Let's also add this to the main readme, and add a changelog entry

File: pkg/deployments/tasks/20221205-veboost-v2/test/test.fork.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    expect(await delegationProxy.getDelegationImplementation()).to.be.eq(delegation.address);
+  });
+
+  it('allows existing boosts to be migrated', async () => {
+    const existingBoosts: string[] = [
+      '0x7f01d9b227593e033bf8d6fc86e634d27aa85568000000000000000000000000',
+      '0xc2593e6a71130e7525ec3e64ba7795827086df0a000000000000000000000000',
+      '0xef9a40f0ce782108233b6a5d8fef08c89b01a7bd000000000000000000000000',
+      '0x0035fc5208ef989c28d47e552e92b0c507d2b318000000000000000000000000',
+    ];
```
Comment (nventuro) [inline pkg/deployments/tasks/20221205-veboost-v2/test/test.fork.ts:65]:
```suggestion
```

These are already declared above

File: pkg/deployments/tasks/20221205-veboost-v2/test/test.fork.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  let task: Task;
+
+  const GOV_MULTISIG = '0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f';
+
+  const existingBoosts: string[] = [
+    '0x7f01d9b227593e033bf8d6fc86e634d27aa85568000000000000000000000000',
+    '0xc2593e6a71130e7525ec3e64ba7795827086df0a000000000000000000000000',
+    '0xef9a40f0ce782108233b6a5d8fef08c89b01a7bd000000000000000000000000',
+    '0x0035fc5208ef989c28d47e552e92b0c507d2b318000000000000000000000000',
+  ];
```
Comment (nventuro) [inline pkg/deployments/tasks/20221205-veboost-v2/test/test.fork.ts:24]: Can we check if this is comprehensive?

Comment (TomAFrench) [reply pkg/deployments/tasks/20221205-veboost-v2/test/test.fork.ts]: I've added a check to ensure that this array is equal in length to the total supply of boosts.

**Review by nventuro — APPROVED** [review]
I've checked that the build info matches commit b61ab2e692093b3e3b58eeeea6ca4df535894bd3.

**Pull request merged**